packages/contract-wrappers/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": ["@0xproject/tslint-config"]
+    "extends": ["@0xproject/tslint-config"],
+    "linterOptions": {
+        "exclude": ["src/contract_wrappers/generated/*"]
```

Comment:
Did this work for you? It did not for me. I was only able to get it working by passing it in as a commandline option. I think we can remove this.

Reply:
It did seem to work